--- entourage/Scenes/Events/EventEditMainViewController.swift
+++ entourage/Scenes/Events/EventEditMainViewController.swift
@@ -535,11 +535,14 @@
 //MARK: - MJAlertControllerDelegate -
 extension EventEditMainViewController: MJAlertControllerDelegate {
     func validateLeftButton(alertTag: MJAlertTAG) {
-        self.dismiss(animated: true)
+        // Le bouton gauche est "Annuler", la popup se ferme toute seule. On ne fait rien d'autre.
     }
     func validateRightButton(alertTag: MJAlertTAG) {
         if alertTag == .Suppress {
             let isAll = selectedRecurrencyPosition == 1
             self.createEvent(applyToAll:isAll)
+        } else if alertTag == .None {
+            // Le bouton droit est "Quitter" pour l'alerte de retour en arriere
+            self.dismiss(animated: true)
         }
     }
